app/views/shared/_client_primary_info_fields.html.erb

```diff
@@ -16,17 +16,19 @@
       <%= f.cfa_select(:timezone, t("general.timezone"), timezone_select_options) %>
     <% end %>
 
-    <% unless is_dropoff %>
+    <% if !is_dropoff && !is_ctc %>
       <%= f.cfa_input_field(:interview_timing_preference, t("hub.clients.show.interview_timing_preference")) %>
     <% end %>
 
-    <%= f.cfa_select(
-            :preferred_interview_language,
-            t("hub.clients.fields.preferred_interview_language"),
-            language_options(only_locales: false),
-            include_blank: true,
-            selected: @form.preferred_interview_language
-    ) %>
+    <% if is_dropoff || !is_ctc %>
```

Comment:
this means it will stay on ctc dropoff right? That seems to match the logic from the ticket, but wanted to make sure im understanding correctly.